node/muneem/app.js

```diff
@@ -0,0 +1,25 @@
+const app = require('muneem')();
+
+app.route({
+  url: "/",
+  to: function(req, res){
+      res.end(200);
+  }
+})
+
+app.route({
+  url: "/user/:id",
+  to: function(req, res){
+      res.write(req.params.id);
+      res.end();
+  }
+})
+
+app.route({
```

Comment:
@amitguptagwl show use `POST` http method

@see https://travis-ci.com/the-benchmarker/web-frameworks/jobs/168545866#L514

Reply:
done